src/Sulu/Bundle/ContentBundle/Content/Types/SingleInternalLink.php

```diff
@@ -92,6 +111,34 @@ public function getTemplate()
         return $this->template;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getContentData(PropertyInterface $property)
+    {
+        $data = $this->loadStructure($property->getValue(), $property->getStructure()->getLanguageCode());
+
+        if (isset($data['content'])) {
+            $content = $data['content'];
+            unset($data['view']);
+            unset($data['content']);
+
+            return array_merge($data, $content);
```

Comment:
did merge the root data so `uuid`, `created`, `authored`, ... are also available in the content type.

Reply:
@sulu/core-team what do you think? in my opinion we should merge this data into view and not into content

Reply:
@wachterjohannes in the navigation this data is also available at the same level as the content.

Reply:
that's right - but in the content variable, there are only `properties` defined in XML ... UUIDs and other configurations (see smart_content) are in the view variable

Reply:
hmm ok but seems not be the same as for `internal_links` there it looks like the same as for the navigation.

Reply:
I am not sure what the result of this statement is, but if there is a `single_internal_link` property on the root level of a template I'd expect to access the `title` of the link in the following way in the twig template:

``` jinja
{content.single_internal_link.title}
```

And the view should only hold the `uuid` of the selected page, since the data that is written in PHPCR should be exposed there.

Resp. it really looks weird to have the `view` data in here and in the `view` twig variable (that's what's happening here, right?)

Reply:
@danrot content does not include the view data. The current data in the template twig file would look like this.

```yml
content:
     title: ...
     single_internal_link:
          # root data
          uuid: ...
          published: ...
          created: ...
          # content
          title: Test
          url: /test
          images: 
                - ...
          ...
view:
     title: ...
     single_internal_link:
          # view
          title: []
          url: []
          images:
              displayOption: left
          ...
```

so in twig the title is accessable as `{{ content.single_internal_link.title }}`

Reply:
@sulu/core-team I think this is the only blocker here, right? I'd say we should decide in a team review what should be happening here, and then merge it. For me the resulting structure as @alexander-schranz has sketched it in the previous comment look good, if everybody is ok with that we could merge this after a rebase.

Reply:
/cc @wachterjohannes @chirimoya